grails-app/i18n/messages.properties

```diff
@@ -180,3 +180,5 @@ sensor.transmitterType.unique=Tag with this serial number already exists and has
 #Property [{0}] of class [{1}] with value [{2}] must be unique
 
 mail.notification.detection.new.subject=AATAMS detections for tags {0}
+
+nonPaginatedMax.warning=Showing maximum of {0} results only, adjust filter or download to different results.
```

Comment:
Esp since I've missed out a word :-)  How about:

"Showing maximum of {0} results only.  Adjust filter, and/or download as CSV file, to see further results."